tinyagentos/cli/taosctl/commands/decisions.py

```
@@ -0,0 +1,138 @@
+def _coerce_value(raw: str):
+    """A multi_select answer is a JSON array; every other answer is the raw
+    string. Only a clearly-bracketed array ([...]) is parsed, so a free-text
+    answer that merely starts with a bracket (or is a JSON object) is left
+    untouched. A bracketed-but-malformed value is a usage error, surfaced
+    cleanly (exit 1) rather than silently forwarded to the server as a string."""
+    text = raw.strip()
+    if text[:1] == "[" and text[-1:] == "]":
+        try:
+            return json.loads(text)
+        except json.JSONDecodeError as exc:
+            raise TransportError(f"--value looks like a JSON array but is invalid: {exc}")
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)